charmhelpers/contrib/hardening/audits/apache.py

```diff
@@ -98,3 +98,8 @@ def _disable_module(module):
     def _restart_apache():
         """Restarts the apache process"""
         subprocess.check_output(['service', 'apache2', 'restart'])
+
+    @staticmethod
+    def is_ssl_enable():
```

Comment:
The method should be named `is_ssl_enabled` vs `is_ssl_enable`.

Reply:
Thanks Billy, I removed the method to 'is_ssl_enabled' and added a unit test for it in tests/contrib/hardening/audits/test_apache_audits.py